cg/meta/workflow/balsamic.py

```diff
@@ -373,6 +373,36 @@ def get_verified_config_case_arguments(
             "normal_sample_name": self.get_normal_sample_name(case_id=case_id),
         }
 
+    def build_sample_id_map_string(self, case_id: str) -> str:
+        """Creates sample info string for balsamic with format lims_id:tumor/normal:customer_sample_id"""
+
+        tumor_sample_lims_id = self.get_tumor_sample_name(case_id=case_id)
+        tumor_string = f"{tumor_sample_lims_id}:tumor:{self.store.sample(internal_id=tumor_sample_lims_id).name}"
+        normal_sample_lims_id = self.get_normal_sample_name(case_id=case_id)
+        if normal_sample_lims_id:
+            normal_string = f"{normal_sample_lims_id}:normal:{self.store.sample(internal_id=normal_sample_lims_id).name}"
+            return ",".join([tumor_string, normal_string])
+        return tumor_string
+
+    def build_case_id_map_string(self, case_id: str) -> str:
+        """Creates case info string for balsamic with format panel_shortname:case_name:application_tag}"""
+
+        case_obj: models.Family = self.store.family(case_id)
+        capture_kit = self.lims_api.capture_kit(case_obj.links[0].sample.internal_id)
+        if capture_kit:
+            panel_shortname = self.store.bed_version(capture_kit).shortname
+        elif self.get_application_type(case_obj.links[0]) == "wgs":
+            panel_shortname = "Whole_Genome"
+        else:
+            panel_shortname = "Undefined"
```

Comment:
Panel short name cannot be "Undefined" in the report. Please use the short name as it is defined here: https://clinical-api.scilifelab.se/admin/bedversion/